# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/sirkull/validator/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./include/RangeMap.h.orig	2005-02-25 19:06:21.234835880 +0100
+++ ./include/RangeMap.h	2005-02-25 19:27:06.715494040 +0100
@@ -54,7 +54,8 @@
   }
 private:
   size_t count_;
-  typename Vector<RangeMapRange<From,To> >::const_iterator ptr_;
+  const RangeMapRange<From,To> *ptr_;
+  // typename Vector<RangeMapRange<From,To> >::const_iterator ptr_;
 };
 
 #ifdef SP_NAMESPACE
